nixos/modules/hardware/ckb-next.nix

```diff
@@ -3,17 +3,22 @@
 with lib;
 
 let
-  cfg = config.hardware.ckb;
+  cfg = config.hardware.ckb-next;
 
 in
   {
-    options.hardware.ckb = {
+    imports = [
+      (mkRenamedOptionModule ["hardware" "ckb" "enable"] ["hardware" "ckb-next" "enable"])
+      (mkRenamedOptionModule ["hardware" "ckb" "package"] ["hardware" "ckb-next" "package"])
+    ];
```

Comment:
Oops, looks like I missed those renames. I personally don't mind if those are kept local to the relevant module, but it looks like most renames are in `nixos/modules/rename.nix`.

I wouldn't hold back the change on account of that, though let's say what you yourself think about it, and if others have a strong opinion about that.